ui/packages/consul-ui/app/components/topology-metrics/card/index.scss

```diff
@@ -75,7 +75,7 @@
   .details {
     padding: 0 12px 12px 12px;
   }
-  div.stats {
+  div.topology-metrics-stats {
```

Comment:
Do you remember why this is in a different file to the rest of the component styles?

Reply:
I was adding the parents from the parent <Card /> component and that's how it ended up here. I've moved it over.